habushu-maven-plugin/src/main/resources/META-INF/plexus/components.xml

```diff
@@ -14,8 +14,8 @@
                         <id>default</id>
                         <phases>
                             <initialize>
```

Comment:
I: just cleaned up the order here as we should make sure poetry is set up before initializing it